mod/log/pkg/phuslu/config.go

```diff
@@ -22,16 +22,67 @@ package phuslu
 
 import "time"
 
+// colours.
+const (
+	reset      = "\x1b[0m"
+	black      = "\x1b[30m"
+	red        = "\x1b[31m"
+	green      = "\x1b[32m"
+	yellow     = "\x1b[33m"
+	blue       = "\x1b[34m"
+	magenta    = "\x1b[35m"
+	cyan       = "\x1b[36m"
+	white      = "\x1b[37m"
+	gray       = "\x1b[90m"
+	lightWhite = "\x1b[97m"
+)
+
 // Config is a structure that defines the configuration for the logger.
 type Config struct {
 	// TimeFormat is a string that defines the format of the time in
 	// the logger.
-	TimeFormat string
+	TimeFormat string `mapstructure:"trace_color"`
+
+	// Colours for the different log levels.
+	TraceColor   string `mapstructure:"trace_color"`
```

Comment:
yeah remove colours and label, too overkill

Reply:
👍